Assets/Scripts/Menus/Options/DevPanel.cs

```diff
@@ -96,13 +114,30 @@ public class DevPanel : MonoBehaviour {
     }
 
     public void CopyToClipboard() {
-        GUIUtility.systemCopyBuffer = GetFlightParams().ToJsonString();
+        var json = GetFlightParams().ToJsonString();
+
+        var jObject = JObject.Parse(json);
+        if ((float)jObject.GetValue("mass") == ShipParameters.Defaults.mass)
+            jObject.Remove("mass");
+        if ((float)jObject.GetValue("inertiaTensorMultiplier") == ShipParameters.Defaults.inertiaTensorMultiplier)
+            jObject.Remove("inertiaTensorMultiplier");
+        if ((float)jObject.GetValue("boostSpoolUpTime") == ShipParameters.Defaults.boostSpoolUpTime) // again remove if boost spool up time has been properly implemented. 
```

Comment:
okay so to be clear this value is added in the json but omitted from export right now and not used because of audio?

Reply:
Yes, it is also hidden from the menu.